g_docs.go

```diff
@@ -1216,3 +1213,24 @@ func generatePathInfo(file *ast.File) (map[string]string, error) {
 
 	return pathInfo, nil
 }
+
+func getPackageName() (string, error) {
+	// Go modules :)
+	f, err := os.ReadFile(modfileName)
+	if err == nil {
+		mf, err := modfile.Parse(modfileName, f, nil)
+		if err == nil {
```

Comment:
can we return the error when the mod file is not valid?

Reply:
No because the whole logic here is:

Use the go module if the project has it, otherwise use gopath. So the gomod being empty or "not found" is possible and not an error.

Reply:
isn't the "not found" state is already taken into account when checking the error of `os.ReadFile(modFileName)`? what I'm curious about here is the `modfile.Parse(modfileName, f, nil)` part. the code reaching the line means that go.mod is already present and found, right?

but on a second note, I just realized that what we do here is parsing the `go.mod` file of the repo that is using `bee generate docs` (re: Doraemon). So, it's not really the responsibility of this tool to check that. Thus, I'm okay with not returning the error here.